src/Majora/Bundle/FrameworkExtraBundle/Controller/RestApiControllerTrait.php

```diff
@@ -161,11 +161,10 @@ protected function assertSubmitedJsonFormIsValid(Request $request, FormInterface
         }
 
         // data camel case normalization
-        $normalizer = $this->container->has('fos_rest.normalizer.camel_keys') ?
-            $this->container->get('fos_rest.normalizer.camel_keys') :
-            null
+        $data = $this->container->has('majora.inflector') ?
+            $this->container->has('majora.inflector')->normalize($data, 'camelcase') :
```

Comment:
$this->container->has('majora.inflector')->normalize

C'est pas "get" plutôt ?